geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegionStats.java

```diff
@@ -1188,11 +1210,51 @@ public void endPutLocal(long start) {
     stats.incLong(putLocalTimeId, delta);
   }
 
+  public long startBucketClear() {
+    incBucketClearsInProgress();
+    return clock.getTime();
+  }
+
+  public void endBucketClear(long start) {
+    long delta = clock.getTime() - start;
+    stats.incLong(bucketClearsId, 1);
+    decBucketClearsInProgress();
+    stats.incLong(bucketClearTimeId, delta);
+  }
+
   public void incPRMetaDataSentCount() {
     this.stats.incLong(prMetaDataSentCountId, 1);
   }
 
   public long getPRMetaDataSentCount() {
     return this.stats.getLong(prMetaDataSentCountId);
   }
+
+  public long getBucketClearCount() {
+    return stats.getLong(bucketClearsId);
+  }
+
+  public void incBucketClearCount() {
+    stats.incLong(bucketClearsId, 1L);
+  }
+
+  public void incBucketClearTime(Long nanoseconds) {
```

Comment:
with the addition of startBucketClear and endBucketClear this method should no longer be needed.

Reply:
done

Reply:
Why does incBucketClearTime still exist? It should be removed